tests/integration/solidity_contracts/PlainOpcodes/test_plain_opcodes.py

```diff
@@ -188,6 +188,32 @@ async def test_should_create_counters(
             ).result.nonce
             assert nonce_final == nonce_initial + count
 
+        @pytest.mark.skip(
+            "TODO: need to fix when there is no return data from the bytecode execution in EVMInstructions.run"
+        )
+        @pytest.mark.parametrize("bytecode", ["0x", "0x6000600155600160015500"])
+        async def test_should_create_contract_with_bytecode_and_no_return_data(
+            self,
+            plain_opcodes,
+            plain_opcodes_deployer,
+            bytecode,
+        ):
+            nonce_initial = (
+                await plain_opcodes.contract_account.get_nonce().call()
+            ).result.nonce
+
+            evm_addresses = await plain_opcodes.create(
+                bytecode=bytecode,
+                count=1,
+                caller_address=plain_opcodes_deployer,
+            )
+            assert len(evm_addresses) == 1
+
+            nonce_final = (
+                await plain_opcodes.contract_account.get_nonce().call()
+            ).result.nonce
+            assert nonce_final == nonce_initial + 1
```

Comment:
I think that we can remove the nonce things since this is not the purpose of this test

Reply:
we should however assert that the deployed bytecode is the expected one

Reply:
Right we don't need the nonce but bytecode should be good 👍